packages/vite-plugin-svelte/src/utils/options.ts

```diff
@@ -195,6 +197,22 @@ function removeIgnoredOptions(options: ResolvedOptions) {
 	}
 }
 
+function setKitOptions(options: ResolvedOptions) {
```

Comment:
it's not actually the Kit options that are being set here, but the compiler options. maybe it should be called `setUsingKitOptions` or something

Reply:
thanks, upated and added a small comment